src/server/scripts/Spells/spell_druid.cpp

```diff
@@ -177,6 +182,38 @@ class spell_dru_bear_form_passive : public SpellScriptLoader
         }
 };
 
+// -50334 - Berserk
+class spell_dru_berserk : public SpellScriptLoader
+{
+    public:
+        spell_dru_berserk() : SpellScriptLoader("spell_dru_berserk") { }
+
+            class spell_dru_berserk_AuraScript : public AuraScript
+        {
```

Comment:
Identation...
Will be nice if you use the new Script Registers to do it like:
https://github.com/TrinityCore/TrinityCore/blob/3.3.5/src/server/scripts/Spells/spell_mage.cpp#L661-L686